### ISSUE TITLE
If stop command not able to shutdown VM on given time limit ask user for force kill

### ISSUE THREAD

**Issue by praveenkumar** [code-ready/crc#162]
Right now stop command is failed to shutdown the domain and error out. we should catch it and ask user if they prefer to kill it.

```
$ crc stop
VM Failed to gracefully shutdown, try the kill command 
```

**Comment by anjannath** [comment]
This should also contain a force flag `--force | -f` so that this can be also performed non-intervention.

**Comment by anjannath** [comment]
Closed in #164
